mentat/code_map.py

```diff
@@ -26,9 +25,7 @@ def get_ctags(
     else:
         ctags_cmd_args.append("--fields=+S")
     ctags_cmd = ["ctags", *ctags_cmd_args, str(abs_file_path)]
-    output = subprocess.check_output(
-        ctags_cmd, stderr=subprocess.DEVNULL, start_new_session=True, text=True
-    ).strip()
+    output = subprocess.check_output(ctags_cmd, stderr=subprocess.DEVNULL, start_new_session=True, text=True).strip()
```

Comment:
There's no actual change here or in the previous line right? Did black formatting change?

Reply:
Yeah I think something is wrong with my formatter...